prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -235,8 +234,8 @@ VirtualHost "{{ $XMPP_DOMAIN }}"
     room_metadata_component = "metadata.{{ $XMPP_DOMAIN }}"
     {{ if $ENABLE_LOBBY }}
     lobby_muc = "lobby.{{ $XMPP_DOMAIN }}"
-    {{ if $ENABLE_RECORDING }}
-    muc_lobby_whitelist = { "{{ $XMPP_RECORDER_DOMAIN }}" }
+    {{ if or $ENABLE_RECORDING $ENABLE_TRANSCRIPTIONS }}
```

Comment:
Let's remove the condition, there is no good reason not to have the whitelist inplace at all times.

Reply:
done